File: C/week3/Guessing_game.c
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>
#include <time.h>
int main()
{
    srand(time(0));
    int randomNumber = rand() % 100 + 1; // random number generate 0-100
    printf("%d\n", randomNumber);

    while (true)
    {
        int guessNumber;
        printf("Enter guess number: ");
        scanf("%d", &guessNumber);
        if (guessNumber == randomNumber)
        {
            printf("Your guess number is right.\n");
            break;
        }
        else if (guessNumber > randomNumber)
        {
            printf("Please, enter smaller number\n");
        }
        else
        {
            printf("Please, enter greater number\n");
        }
    }

    return 0;
}

File: Competitive Programming/Basic C C++ Python Java/Basic Python/guessingGame.py
import random
from random import randint

n = int(input("Enter a number: "))
# guessNumber = random.randint(1, 6)
# guessNumber = random.random()*100
guessNumber = int(random.random()*100)
if n == guessNumber:
    print("you are won!")
else:
    print("you loss")
    print("Random number is: ", guessNumber)
